impeller/renderer/backend/vulkan/context_vk_unittests.cc

```diff
@@ -143,5 +143,35 @@ TEST(ContextVKTest, CanCreateContextWithValidationLayers) {
   ASSERT_TRUE(capabilites_vk->AreValidationsEnabled());
 }
 
+// In Impeller's 2D renderer, we no longer use stencil-only formats. They're
+// less widely supported than combined depth-stencil formats, so make sure we
+// don't fail initialization if we can't find a suitable stencil format.
+TEST(CapabilitiesVKTest, ContextInitializesWithNoStencilFormat) {
```

Comment:
You should be able to bring back the previous test case using `ScopedValidationDisable`

Reply:
Ah forgot about this! Done.